File: week3/Choi/Week3/Whip.cs
namespace Week3
{
    public class Whip : CondimentDecorator
    {
        public Beverage m_Beverage;

        public Whip(Beverage ver)
        {
            m_Beverage = ver;

        }

        public override double Cost()
        {
            return 0.1f + m_Beverage.Cost();
        }

        public override string GetDescription()
        {
            return m_Beverage.GetDescription() + " 휘핑 크림";
        }
    }
}

File: week3/younggeun0/Whip.java
package week3.younggeun0;

public class Whip extends CondimentDecorator {
    Beverage beverage;

    public Whip(Beverage beverage) {
        this.beverage = beverage;
    }

    @Override
    public String getDescription() {
        return beverage.getDescription() + ", 휘핑 크림";
    }

    @Override
    public double cost() {
        return beverage.cost()+0.10;
    }
}